BattleNetwork/bnCharacter.cpp

```
@@ -17,6 +17,9 @@
 #include "bnCardAction.h"
 #include "bnCardToActions.h"
 
+// All statuses which should prevent Character from taking actions
+constexpr Hit::Flags blockingStatuses = Hit::stun | Hit::freeze | Hit::bubble | Hit::drag;
```

`constexpr const Hit::Flags` ...

The keyword `constexpr` is akin to Zig's `comptime` keyword. It was added much later to the C++ spec and allows the term to be used during compile-time evaluations. However, after compile-time, it can be mutated. Also, if a context marked `constexpr`, like a function body, is ever used ambiguously during compile-time, then the compiler silently drops `constexpr`. 

TL;DR

`constexpr const` for compile-time constants.

---

Why is it **const**expr if it's not already const? Will add.

---

Added in bd315f0a7313957a26cc8f9f81509126d8222de7.